### PR TITLE
add hub-lib to pipeline

### PR DESCRIPTION
**Pull request by dominiksteiner**

### CODE REVIEW

**Comment by dominiksteiner** [comment]
@jodonnell-broadsoft ready to review

File: tasks/pipeline.js [hunk 1]
```diff
@@ -28,6 +28,7 @@ var jsFilesToInject = [
   'bower_components/angular-cookies/angular-cookies.js',
   'bower_components/underscore/underscore-min.js',
   'bower_components/moment/moment.js',
+  'bower_components/hub-lib/app/app.js',
```
Comment (jodonnell-broadsoft) [inline tasks/pipeline.js]: Ah. I see what you are doing here now. Maybe we can call about this one so I can better understand :)

**Comment by jodonnell-broadsoft** [comment]
Looks fine to me now but lets use something other than app.js to describe this module.

**Comment by dominiksteiner** [comment]
@jodonnell-broadsoft renamed to hubLib/cookie.js

**Comment by jodonnell-broadsoft** [comment]
sweet. Thanks

**Pull request merged**